packages/frontend/public/index.html

```diff
@@ -7,8 +7,9 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Take control of your online presence with a personal .fil domain powered by Spheron. Our easy-to-use platform lets you create and manage your domain with just a few clicks"
     />
+    <meta property="og:image" content="./fns-meta.png" />
```

Comment:
check the frontend repo and add all the other params as well - https://github.com/spheronFdn/frontend/blob/main/public/index.html#L22-L24